### PR TITLE
feat: add ToggleButton.Row component

### LINKED ISSUE #1168: Add a `ToggleButtonRow` component
We have a `ToggleButtonGroup` component which is supposed to provide a way to manage multiple toggle buttons from one place, similar to `RadioButtonGroup`. However, when grouping toggle buttons, it maybe desirable to group them in a row and remove left/right borders of adjacent buttons.

The `ToggleButtonGroup` confusingly adds some styles for borders, but doesn't add style for a row. We should add a `ToggleButtonRow` component which does this and remove all style related code from `ToggleButtonGroup`. The `ToggleButtonRow` component could include `ToggleButtonGroup` by default.

### PR DESCRIPTION
**Pull request by Trancever**
### Motivation

Fix #1168

### CODE REVIEW

File: src/components/ToggleButton/ToggleButtonGroup.tsx [hunk 1]
```diff
@@ -68,51 +67,10 @@ class ToggleButtonGroup extends React.Component<Props> {
...
-                  : styles.middle,
-                // @ts-ignore
-                child.props.style,
-              ],
-            });
-          }
-
-          return child;
-        })}
+        <View style={style}>{children}</View>
```
Comment (satya164) [inline src/components/ToggleButton/ToggleButtonGroup.tsx]: I think this wrapper view can be removed

Comment (Trancever) [reply src/components/ToggleButton/ToggleButtonGroup.tsx]: Ok, then we don't want to accept style prop, right?

Comment (satya164) [reply src/components/ToggleButton/ToggleButtonGroup.tsx]: Yea

Comment (Trancever) [reply src/components/ToggleButton/ToggleButtonGroup.tsx]: Fixed :)

File: src/components/ToggleButton/ToggleButtonRow.tsx [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+  /**
+   * React elements containing toggle buttons.
+   */
+  children: React.ReactNode;
+  style?: StyleProp<ViewStyle>;
+};
+
+/**
+ * Toggle row renders buttons in a row and applies predefined styles.
+ *
```
Comment (satya164) [inline src/components/ToggleButton/ToggleButtonRow.tsx]: nit: remove extra new line

File: src/components/ToggleButton/ToggleButtonRow.tsx [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+  value: string;
+  /**
+   * React elements containing toggle buttons.
+   */
+  children: React.ReactNode;
+  style?: StyleProp<ViewStyle>;
+};
+
+/**
+ * Toggle row renders buttons in a row and applies predefined styles.
```
Comment (satya164) [inline src/components/ToggleButton/ToggleButtonRow.tsx]:
```suggestion
 * Toggle button row renders a group of toggle buttons in a row.
```

**Pull request merged**